Source/WPEFramework/Controller.cpp

```diff
@@ -301,7 +301,7 @@ namespace Plugin {
 
         ASSERT(_pluginServer != nullptr);
 
-        if ((basecallsign.empty() == false) && (basecallsign.empty() == false) && (basecallsign != controllerName) && (newcallsign != controllerName)) {
+        if ((basecallsign.empty() == false) && (basecallsign != controllerName) && (newcallsign != controllerName)) {
```

Comment:
I believe the idea was to make sure that neither basecallsign nor newcallsign is an empty string.